Doc/source/designspaceLib/index.rst

```diff
@@ -120,6 +120,34 @@ any of the UFOs. If the lib key is empty or not present in the Designspace, all
 glyphs should be exported, regardless of what the same lib key in any of the
 UFOs says.
 
+public.fontInfo
+-----------------------
+
+This lib key, when included in the ``<lib>`` element inside an ``<instance>`` or 
+``<variable-font>`` tag, allows for direct manipulation of font info data in 
+instances and variable fonts. The lib value must follow the 
+`UFO3 fontinfo.plist specification <https://unifiedfontobject.org/versions/ufo3/fontinfo.plist/>`_,
+and should functionally appear to be a property list dictionary with the same 
+structure as the ``fontinfo.plist`` file in a UFO.
+
+
+All font info items in the UFO fontinfo.plist specification should be able to 
+be defined in the ``public.fontInfo`` lib. Checking validity of the data using 
+``fontTools.ufoLib.validators`` is recommended but not required. 
+
+A tool generating the font should copy the font info directly from the 
+``public.fontInfo`` lib item to the instance or the variable font, superceding 
```

Comment:
If I am not mistaken, the correct English spelling should be "supersede" with the "s", not "supercede" with "c"